src/libxively/io/fs/posix/xi_fs_posix.c

```diff
@@ -6,6 +6,58 @@
 
 #include "xi_fs_header.h"
 #include "xi_macros.h"
+#include "xi_bsp_io_fs.h"
+
+const size_t xi_fs_buffer_size = 1024;
+
+xi_state_t xi_fs_bsp_io_fs_2_xi_state( xi_bsp_io_fs_state_t bsp_state_value )
```

Comment:
thx for moving this function! Could you please put this function into a file avoiding the duplication in files xi_fs_memory.c and xi_fs_posix.c? I know it is planned to be removed but still who knows how long it'll exist? It may go out of sync for mem and file implementation if two versions exist. Thanks.